spring-web-modules/spring-mvc-file/src/test/java/com/baeldung/file/CustomMultipartFileUnitTest.java

```diff
@@ -0,0 +1,24 @@
+package com.baeldung.file;
+
+import java.io.IOException;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.mock.web.MockMultipartFile;
+
+public class CustomMultipartFileUnitTest {
+
+    @Test
+    public void whenProvidingByteArray_thenMultipartFileCreated() throws IOException {
```

Comment:
junit5 jupiter could be without `public`
```suggestion
    void whenProvidingByteArray_thenMultipartFileCreated() throws IOException {
```

Reply:
Same applies to the class modifier, so please remove the public modifier.